storage/Devices/LvmVgImpl.cc

```diff
@@ -520,4 +520,102 @@ namespace storage
 	return lhs->get_vg_name() < rhs->get_vg_name();
     }
 
+
+    void
+    LvmVg::Impl::add_dependencies(Actiongraph::Impl& actiongraph) const
+    {
+	// First, all the operations removing or shrinking LVs
+	vector<Actiongraph::Impl::vertex_descriptor> decrease_lv_actions;
+	// Then a sync point
+	Action::Base* decrease_lv_sync = new Action::Create(get_sid(), true);
```

Comment:
My code does not longer use sync points. But still I think we should improve this (in a different pull request).

@aschnell What about removing the `only_sync` from `Action::Base` and creating a `Action::Noop` class for syncing purposes? I found it less hacky and easier to debug problems (since `Action::Noop` would have a meaningful implementation of `text()`).